doc/source/getting_started/install.rst

```diff
@@ -307,7 +307,7 @@ Dependency                                             Minimum Version    pip ex
 `PyTables <https://github.com/PyTables/PyTables>`__    3.8.0              hdf5             HDF5-based reading / writing
 `zlib <https://github.com/madler/zlib>`__                                 hdf5             Compression for HDF5
 `fastparquet <https://github.com/dask/fastparquet>`__  2024.2.0           -                Parquet reading / writing (pyarrow is default)
-`pyarrow <https://github.com/apache/arrow>`__          10.0.1             parquet, feather Parquet, ORC, and feather reading / writing
+`pyarrow <https://github.com/apache/arrow>`__          12.0.1             parquet, feather Parquet, ORC, and feather reading / writing
```

Comment:
so this is for the "Other data sources" section.

As is now also improves performance for a default dtype, should also add to "Performance dependencies (recommended)" section or will this be done in another PR, #61722?

Reply:
Good point, we indeed still have to update our installation guidelines to more prominently recommend installing pyarrow. Will open a separate issue about that (currently tracked it in the list of todo items in https://github.com/pandas-dev/pandas/issues/54792)